### PR TITLE
Update orchestrator to support multiple index names

### PR DESCRIPTION
**Pull request by chriskurzeja**

### CODE REVIEW

File: src/__tests__/orchestrator.integration.spec.ts [hunk 1]
```diff
@@ -69,15 +69,18 @@ describe('elastic orchestrator', () => {
 
     let client: Client
 
-    const createBulkPayload = (count: number) =>
+    const createBulkPayload = (
+      count: number,
+      bodyGenerator: (value: number) => object = value => ({
```
Comment (ivoreis) [inline src/__tests__/orchestrator.integration.spec.ts:74]: Do you think we can extract this type into a type variable? I'd make the code more readable.

File: src/__tests__/orchestrator.integration.spec.ts [hunk 2]
```diff
@@ -165,6 +168,35 @@ describe('elastic orchestrator', () => {
...
     })
 
+    it('will trigger a reindex from the index with the same name for a new index', async () => {
+      const existingConfig = ['index-00001.json', 'another-index-00000.json']
+      await manageIndices(client, existingConfig, indexConfigFileFolder)
+
+      const indices = await getExistingIndices(client)
+      const latestIndex = await getMostRecentIndex(
+        client,
+        indices.filter(index => index.startsWith('another-index-')),
```
Comment (ivoreis) [inline src/__tests__/orchestrator.integration.spec.ts:178]: I guess it doesn't make a difference but we might want to exclude the last -, so we only include what is considered to be the `index name`

```suggestion
        indices.filter(index => index.startsWith('another-index')),
```

File: src/__tests__/orchestrator.integration.spec.ts (hunk 1 above)
Comment (chriskurzeja) [reply src/__tests__/orchestrator.integration.spec.ts]: 👍

File: src/__tests__/orchestrator.integration.spec.ts [hunk 3]
```diff
@@ -165,6 +170,35 @@ describe('elastic orchestrator', () => {
...
+      await manageIndices(
+        client,
+        [...existingConfig, 'another-index-00001.json'],
+        indexConfigFileFolder,
+      )
+
+      const oldIndexDocuments = await getIndexDocuments('another-index-00000')
+      const newIndexDocuments = await getIndexDocuments('another-index-00001')
+
+      expect(newIndexDocuments).toEqual(oldIndexDocuments)
```
Comment (ivoreis) [inline src/__tests__/orchestrator.integration.spec.ts:199]: Just to cover all basis we should ensure/assert that `index` and `another-index` are actually different, WDYT?

Comment (chriskurzeja) [reply src/__tests__/orchestrator.integration.spec.ts:199]: Sure.

**Pull request merged**